src/tests/ftest/util/scrubber_utils.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2021 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from logging import getLogger
+from ClusterShell.NodeSet import NodeSet
+from telemetry_utils import TelemetryUtils
+
+
+class ScrubberUtils(TelemetryUtils):
+    def __init__(self, dmg, servers):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class 'TelemetryUtils' is not called